CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# v10.0.0 (2025-07-25)
+
+## OS Changes
+- Deprecate wicked package ([#560])
+- Fix file descriptor leak in apiserver exec ([#595])
+- Add release subpackage to enable zram-backed swap ([#590])
+
+### Third Party Package Updates
+- Update cni, cni-plugins, libaudit, libbpf, libdevmapper, libglib, and libncurses ([#600])
+
+## Orchestrator Changes
+### Kubernetes
```

Comment:
I think the commits from https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/576 that add support for kubelet configuration drop-ins could use their own bullet:

```
* support extending kubelet configuration via drop-in files ([#576])
```